### DISCUSSION TITLE
Compare two dataset runs side by side

### DISCUSSION THREAD

**Discussion by dmaslov** [langfuse/langfuse#2883] [Ideas]
Hello. I'm wondering if it is possible to do something similar in Langfuse as shown in the screenshot. I am trying to find a tool that is able to do both trace monitoring and experiment utilization.

<img width="1345" alt="image" src="https://github.com/user-attachments/assets/40bbdd34-9759-41d5-97e6-55d1830e50ff">

Thanks in advance

**Comment by marcklingen** [comment]
Thanks for raising this. I just moved this thread to "ideas".

This is high up on our backlog for datasets. We will keep you in the loop here once this is released.

**Comment by marcklingen** [comment]
This was released late last year, forgot to update this thread. Thanks again for raising this.
